content/en/_index.html

```diff
@@ -124,61 +116,458 @@
         src="https://raw.githubusercontent.com/cdfoundation/artwork/master/jenkinsx/stacked/color/jenkinsx-stacked-color.png">
       <div class="tag-line">Accelerate Your Continuous<br/> Delivery on Kubernetes</div>
       <p>
-        <a href="/docs/getting-started/" class="btn btn-lg bg-primary text-light my-4">Get started</a>
+        <a href="/docs/getting-started" class="btn btn-lg bg-primary text-light my-4">Get started</a>
       </p>
     </div>
   </section>
-  <section class="py-5 bg-light">
+  <!-- <section class="py-5">
```

Comment:
What's all this? It is even commented out, right?

Reply:
Please leave that as is, it is future content for another PR.